### PR TITLE
README.md doc improvements

### PR DESCRIPTION
**Pull request by samuelgmartinez**
I know that all this information is already on the doc string of the ruby files but would be useful to have it in the project main page.

on top of that I've added some tips on the configuration and what means every property and how to deal with the authentication.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,40 @@ input {
...
+    * **required**: true
+* `region`: The AWS region name for Kinesis, DynamoDB and Cloudwatch (if enabled)
+    * **required**: false
+    * **default value**: `us-east-1`
+* `checkpoint_interval_seconds`: How many seconds between worker checkpoints to DynamoDB. A low value ussually means lower message replay in case of node failure/restart but it increases CPU+network ussage (which increases the AWS costs).
+    * **required**: false
+    * **default value**: `60`
+* `metrics`: Worker metric tracking. By default this is disabled, set it to "cloudwatch" to enable the cloudwatch integration in the Kinesis Client Library. 
+    * **required**: false
+    * **default value**: `60`
```
Comment (codekitchen) [inline README.md]: copy/paste error, should be `nil`

**Comment by codekitchen** [comment]
great stuff, thanks

**Comment by samuelgmartinez** [comment]
Whopsie! we overlooked that when my peer and I reviewed it. Fixed! 

And thanks to you. We are deploying this plugin to production this week. Works really nice :+1:

**Comment by codekitchen** [comment]
and merged, thank you

**Pull request merged**